examples/src/demos/Friction.tsx

```diff
@@ -1,33 +1,74 @@
-import React, { useState } from 'react'
-import { Canvas } from '@react-three/fiber'
-import type { BoxProps, PlaneProps, Triplet } from '@react-three/cannon'
 import { Physics, useContactMaterial, usePlane, useBox } from '@react-three/cannon'
 import { OrbitControls } from '@react-three/drei'
+import { Canvas } from '@react-three/fiber'
+import { useState } from 'react'
+
+import type { BoxProps, PlaneProps } from '@react-three/cannon'
 
 const materialColors = {
-  ground: '#9b7653',
-  slippery: 'royalblue',
-  rubber: 'darkgrey',
   bouncy: 'yellow',
   box: '#BB8E51',
+  ground: '#9b7653',
+  rubber: 'darkgrey',
+  slippery: 'royalblue',
```

Comment:
@bjornstar : I notice from your readability edits you prefer sorted object keys. Would adding a linting rule such as https://eslint.org/docs/rules/sort-keys help make this clear to contributors?

Reply:
That's a good idea.